### ISSUE TITLE
Hey Falconne

### ISSUE THREAD

**Issue by Danomano** [Falconne/ImprovedWorkbenches#21]
Do you have a connection or something with falcons? The Birds? That's why your name's Falconne right?
Anyway dude, I'm a big fan of Rimworld, great game! I'm playing it with the HardcoreSK mod. Here's the thing; I need you to remake the colony manager mod made by fluffy. Do it bro. Specifically I need you to remake the construction tab of that mod. Hey cheers man, I've been playing Rimworld for a solid 1.30 hours a day, I'd have a blast if you could do this. Thanks again and bye bye.

**Comment by Falconne** [comment]
Fluffy's mods are in active development so if you need a new feature best ask for that in their Ludeon forum thread.

What does the construction tab do?

**Comment by Danomano** [comment]
Fuck Fluffy.

**Comment by Falconne** [comment]
If you have specific feature suggestions to improve this mod it's best to make them at https://ludeon.com/forums/index.php?topic=33083.

As for Colony Manager's Production Tab (which I think is what you're referring to?) I don't think there is a big drive to reimplement that as most of that functionality is now provided for by this mod (especially when you use linked bills). The "overview" functionality that the Production Tab provided, while useful, would be a fair bit of work to create and will need constant maintenance. The mod will need to reimplement nearly all the functionality carried out by vanilla workbenches (such as recipe parsing). It also becomes confusing for people if a background "manager" is adding single bills to workbenches.

Basically, the coding for it is a nightmare that no one is really going to be too keen to take on.

**Comment by Danomano** [comment]
Dude you're lame as hell.
